### PR TITLE
Vox raiders radio pt 2

### PR DESCRIPTION
**Pull request by jakmak6**
https://github.com/vgstation-coders/vgstation13/pull/15389 old pr remade because git

As it stands now vox raiders use the same radios that the nukies use allowing for them to hear station freq. However since vox cant understand galcomm the station freqs just clog up their radios. So i gave the vox their own radio frequency, own radios, own all in one server box (same one nukies have) and inter z level functionality while removing the ability for them to hear station radio frequencies.

Also be noted i added other stuff to the vox. ~~3~~ 6 extra new radios, ~~3 agent IDs with trader access and trader sprite~~, and a single station bounced radios so they can negotiate with crew

tested 95%. the allinone boxes are independent and do not affect the frequencies of the other box or station freq, radio channel works fine and does not direct all crew frequencies to the vox

### CODE REVIEW

File: code/game/machinery/telecomms/machine_interactions.dm [hunk 1]
```diff
@@ -319,7 +319,7 @@
 				if(newfreq && canAccess(usr))
 					if(findtext(num2text(newfreq), "."))
 						newfreq *= 10 // shift the decimal one place
-					if (newfreq != SYND_FREQ)
+					if(newfreq == SYND_FREQ || newfreq == RAID_FREQ)
```
Comment (DamianX) [inline code/game/machinery/telecomms/machine_interactions.dm]: I'm pretty sure you need to wrap this in a NOT.

**Comment by SonixApache** [comment]
*again*?

**Comment by jakmak6** [comment]
because in raiders working with 5 idiots with completley uninteligible names while your chatbox is being spammed with multi coloured garble makes it ultra easy to cooperate! That and if a admin wants to do a bus with both nukies and raiders he wont need to worry about their radio channels conflicting

**Comment by jakmak6** [comment]
also the colour for the channel i used was brown (same as the nukie freq) but i was wondering if i should change it to purple/pink (vox pigin colour) or any other colour

**Comment by Exxion** [comment]
For the love of god don't make it the same color as Pidgin

**Comment by SonixApache** [comment]
seperate radio chat tab when

**Comment by CptWad** [comment]
ffffffucking merging conflicting map prs

**Pull request merged**